src/components/.storybook/ColorSwatch/color-swatch.scss

```diff
@@ -1,4 +1,4 @@
-@import '../../../styles/colors';
+@use '../../../styles/colors';
```

Comment:
We were using a library to make imports work a certain way, but it's no longer needed in latest sass update